setup.py

```diff
@@ -36,18 +38,41 @@
 ]
 
 
-def compile_summary_protobuf():
-    proto_paths = ["smdebug/core/tfevent/proto"]
-    cmd = "set -ex && protoc "
-    for proto_path in proto_paths:
-        proto_files = os.path.join(proto_path, "*.proto")
-        cmd += proto_files + " "
-        print("compiling protobuf files in {}".format(proto_path))
-    cmd += " --python_out=."
-    return os.system(cmd)
+@contextlib.contextmanager
+def remember_cwd():
+    """
+    Restore current directory when exiting context
+    """
+    curdir = os.getcwd()
+    try:
+        yield
+    finally:
+        os.chdir(curdir)
+
+
+def scan_git_secrets():
+    from subprocess import check_call
```

Comment:
What is the behavior if the command executed by `check_call` fails? Does it throw an error?